functional-tests/build.gradle

```diff
@@ -13,11 +13,11 @@ ext {
     drivers = ["firefox", "firefoxHeadless", "chrome", "chromeHeadless","ie","edge","safari"]
 
     ext {
-        groovyVersion = '2.5'
+        groovyVersion = '2.5.0'
```

Comment:
This should resolve the current failures.

2.5.4 exists, but we can't upgrade to it yet because spock only supports 2.5.0 at the moment.  That was an issue I found that resulted in reporting to error out (though the tests could still run).

Reply:
I had to use 2.5.1 on my end, but I might have a different configuration.

Reply:
I tested 2.5.1 on my end, and everything works.  So I updated the PR with that version.

Added a comment as well, because anything pat 2.5.1 results in spock reporting to fail the test execution